### PR TITLE
exclude react-globalize files from webpack transforms

### PR DESCRIPTION
**Pull request by alunny**
Depends on rxaviers/globalize-webpack-plugin#9 being merged and published as 0.3.0 :)

### CODE REVIEW

File: util.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  isReactGlobalizeModule: function(filepath) {
+    filepath = filepath.split( "/" );
+    var i = filepath.lastIndexOf("react-globalize");
+    // 1: path should contain "globalize",
```
Comment (rxaviers) [inline util.js]: Comment needs update for "`..contain "react-globalize"`.

**Comment by alunny** [comment]
@rxaviers now that the globalize-webpack-plugin change is in, I'm going to merge this and publish a new release.

Since it changes the behavior of the module, I'll give it a minor version bump to 0.4.0.

**Comment by rxaviers** [comment]
:+1:


## 

+55 (16) 98138-1583, skype: rxaviers
http://rafael.xavier.blog.br

**Comment by rxaviers** [comment]
@alunny, this change broke some of my applications (that used react-globalize solely). Please, could you refresh my memory, was it motivated by https://github.com/rxaviers/globalize-webpack-plugin/issues/8? If so, it seems like the correct approach is going to be handling default messages in Globalize, not by this change. What do you think? Thanks.

PS: For clarity, the change in globalize-webpack-plugin is totally fine. I think the problem is to use the filter to skip react-globalize files.

**Comment by alunny** [comment]
Do you have react-globalize in your vendor bundle (in the webpack config), or globalize in the vendor bundle and react-globalize in the app bundle?

I think the filter needs to be there if react-globalize is in the vendor bundle, but causes problems if it's in the app bundle.

**Comment by rxaviers** [comment]
I've created a simple example (which probably needs simplifications and improvements) here #12, which shows your update didn't cause any problems. So, please ignore my comment.

PS: I have both globalize and react-globalize in vendors in my app, but I still don't know what's causing the failure. Investigating.

**Comment by alunny** [comment]
Thanks @rxaviers - I'll have a look at #12 and let you know of any suggestions.

**Comment by rxaviers** [comment]
Thank you.

**Comment by rxaviers** [comment]
@alunny, FYI I'm finally back to this and so far I've noticed #14.

**Pull request merged**